interim-jan-2021/slides-directories.md

```diff
@@ -0,0 +1,30 @@
+# Scenario 3: Directories of Encrypted Resolvers
+
+Three Parties:
+* Publisher: Curates a list of distinct resolvers
+* Client: Fetches the list from a trusted source
+* Resolvers: Identified in the list, ready for access by the client
+------------------------------------------------
+# Example use cases
+
+* A browser vendor wants to provide users with a list of resolvers to consider, curated by a trusted third party
+* An OS vendor wants to keep its list of trusted resolvers current without requiring a software update
+* A user wants to choose a resolver from a list offered by a network operator who they trust
+------------------------------------------------
```

Comment:
From @jim-add

```suggestion
* A user wants to select a resolver from a list determined by a third party (eg from an app-supplied list)
------------------------------------------------
```

Reply:
@jim-add, what's the distinction you see between this use case and the "browser" bullet above?

Reply:
As agreed, I've combined these into one line.